src/probabilities_estimators/permutation_ordinal/spatial_permutation.jl

```diff
@@ -21,13 +23,34 @@ stencil = CartesianIndex.([(0,1), (1,1), (1,0)])
 est = SpatialSymbolicPermutation(stencil, x)
 ```
 
+or equivalently
+
+```julia
+data = [rand(50, 50) for _ in 1:50]
+x = data[1] # first "time slice" of a spatial system evolution
+stencil = [1 1; 1 1]
```

Comment:
From further down the docstring: "... no offset (meaning the pixel itself) is always included automatically". How is this handled in the case of integer array stencils, and in the case of `extent` and `lag` stencils?

Does it mean, for example for the 2D case, that the stencil `[0 1; 1 1]` would not be possible?

Reply:
Your example would be possible, the offset doesn't have to be in the upper left corner of the matrix. The way I generate the stencils from the input array, I make sure to translate them in a way that the "no offset" coordinate can be included. Your example would be translated into `CartesianIndex.([(0,1), (-1,1)])` (plus the `(0,0)` that will later be added).

Reply:
> Your example would be possible, the offset doesn't have to be in the upper left corner of the matrix. The way I generate the stencils from the input array, I make sure to translate them in a way that the "no offset" coordinate can be included. Your example would be translated into CartesianIndex.([(0,1), (-1,1)]) (plus the (0,0) that will later be added).

Excellent! Perhaps then the docstring should say that if using integer arrays as stencils, then the "pixel itself is included" part doesn't apply - instead, what you demand is what you get - or something along those lines?